clustered/server/entity/src/main/java/org/ehcache/clustered/server/internal/messages/EhcacheSyncMessageCodec.java

```diff
@@ -154,7 +154,7 @@ public EhcacheSyncMessageCodec(ResponseCodec responseCodec) {
               });
           }
         })
-      .int32(MESSAGE_TRACKER_SEGMENT_FIELD, syncMessage.getSegmentId());
+      .int32(MESSAGE_TRACKER_SEGMENT_FIELD, 0);
```

Comment:
Probably worth a comment about backward compatibility here.

Reply:
I think we should just remove.  A new message has been added so new servers can no longer sync old servers.  Rolling upgrade is still possible.